components/SearchBar.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useRef } from "react";
+
+export default function SearchBar({ inputValue, setInputValue }) {
```

Comment:
I think this naming would be more clear now it is in a separate component

```suggestion
export default function SearchBar({ value, onChange }) {
```